packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -369,57 +370,75 @@ export default class GoogleCalendarService implements Calendar {
     timeMin: string;
     timeMax: string;
     items: { id: string }[];
-  }): Promise<calendar_v3.Schema$FreeBusyResponse> {
+  }): Promise<EventBusyDate[] | null> {
```

Comment:
The only thing that changed in the `getCacheOrFetchAvailability` function is that we return now`EventBusyDate[]` instead of `calendar_v3.Schema$FreeBusyResponse`